db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,22 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
+      t.string   :ssh_private
+      t.string   :ssh_public
+      t.string   :ssh_user
+      t.string   :rhsm_user
+      t.string   :rhsm_pass
+      t.string   :rhsm_sku
+      t.string   :rhsm_pool_id
```

Comment:
there should also probably be fields for rhsm_server (satellite server to subscribe to), and rhsm_activation_key (which will need to be added to the rhel_subscribe role to support).